include/core/Color.hpp

```diff
@@ -73,6 +74,23 @@ struct Color {
 		return components[idx];
 	}
 
+    Color operator+(const Color &p_color) const;
+    void operator+=(const Color &p_color);
+
+    Color operator-() const;
+    Color operator-(const Color &p_color) const;
+    void operator-=(const Color &p_color);
+
+    Color operator*(const Color &p_color) const;
+    Color operator*(const real_t &rvalue) const;
+    void operator*=(const Color &p_color);
+    void operator*=(const real_t &rvalue);
+
+    Color operator/(const Color &p_color) const;
+    Color operator/(const real_t &rvalue) const;
+    void operator/=(const Color &p_color);
+    void operator/=(const real_t &rvalue);
```

Comment:
Indentation here seems to be different than all other lines

Reply:
It's a big copy/paste from Godot's codebase, and I use clang-format... strange it didnt get picked up